# HG changeset patch
# User pm
# Date 1526645396 -7200
#      Fri May 18 14:09:56 2018 +0200
# Node ID 1662347bb9f4e98f978e697a5499053cd60e41f1
# Parent  b276bcbf3b585cfc20bdd44409e5f87b97cd8485
Ticket mantis 0003038: Menu application de Thunderbird

diff --git a/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul b/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul
--- a/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul
+++ b/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul
@@ -2134,6 +2134,8 @@
             <menuitem id="appmenu_addressBook"
                       label="&addressBookCmd.label;"
                       key="key_addressbook"
+                      image="chrome://messenger/skin/addressbook/icons/addrbook.png"
+                      class="menuitem-iconic"
                       oncommand="toOpenWindowByType('mail:addressbook', 'chrome://messenger/content/addressbook/addressbook.xul');"/>
             <menuseparator id="devToolsSeparator"/>
             <menuitem id="appmenu_openSavedFilesWnd"
@@ -2221,13 +2223,12 @@
                         label="&openHelp.label;"
                         key="key_openHelp"
                         oncommand="openSupportURL();"/>
+              <menuitem id="appmenu_releaseNotes"
+                        label="&releaseCmd2.label;"
+                        oncommand="openFormattedURL('app.releaseNotesURL');"/>
               <menuitem id="appmenu_whatsNew"
                         label="&whatsNewCmd.label;"
                         oncommand="openWhatsNew();"/>
-              <menuitem id="appmenu_releaseNotes"
-                        label="&releaseCmd2.label;"
-                        oncommand="openFormattedURL('app.releaseNotesURL');"/>
-              <menuseparator/>
               <menuitem id="appmenu_troubleshootingInfo"
                         label="&helpTroubleshootingInfo.label;"
                         oncommand="openAboutSupport();"/>
@@ -2236,6 +2237,9 @@
                   label="&helpSafeMode.label;"
                   oncommand="safeModeRestart();"/>
               <menuseparator/>
+              <menuitem id="appmenu_checkForUpdates"
+                        label="&updateCmd.label;"
+                        oncommand="checkForUpdates();"/>
               <menuitem id="appmenu_about"
                   label="&aboutMenuCmd2.label;"
                   oncommand="openAboutDialog();"/>
